src/AlgAssAbsOrd/Ideal.jl

```diff
@@ -1166,7 +1166,7 @@ end
 Given an order $O$ and a prime $p$, it returns the radical of the ideal
 generated by $p$.
 """
-function pradical(O::AlgAssAbsOrd, p::Int)
+function pradical(O::AlgAssAbsOrd, p::Union{Int, ZZRingElem})
```

Comment:
```suggestion
function pradical(O::AlgAssAbsOrd, p::IntegerUnion)
```
Why not use `IntegerUnion`?